src/angular/header-lean/header.scss

```diff
@@ -214,6 +214,10 @@
 
   > svg {
     width: calc(#{sbb.pxToRem(80)} * var(--sbb-scaling-factor));
+
+    :where(.sbb-off-brand-colors) & path:first-child {
```

Comment:
Should we maybe add a class to our default logo and add it to the selector? This way, if a consumer uses a different svg it does not have side effects.